taccsite_cms/templates/guides/portal_technology.html

```diff
@@ -45,115 +49,118 @@ <h2>High-level Architecture</h2>
-    <li><strong>Community Data</strong> view displays non-curated user-contributed data for public use.</li>
+    <li><em>My Data</em> systems can exist on Work or Scratch. These systems host data that is only accessible to the authenticated portal user. This is the default storage system for all application output data. Within the MyData space, users can organize, manipulate, view, and copy the data to other storage resources.</li>
+    <li><em>Shared Workspaces</em> display any workspaces the user has created or that another user has shared with the user. Within this context, files associated with a specific project can be manipulated, tagged, organized, and published through the Data Curation workflow, where publications automatically receive a DOI.</li>
+    <li><em>Community Data</em> displays non-curated user-contributed data shared with all users who have access to the portal.</li>
+    <li><em>Public Data</em> exposes curated data for making research findings publicly available.</li>
+    <li><em>Cloud storage</em> systems expose third-party storage resources that have been integrated into the portal user's account (i.e. - Google Drive, Box, UT Box).
   </ul>
 
   <h3>Applications</h3>
 
-  <p>APPS are executable code available for invocation through Agave’s Jobs service on a specific execution system. If a single code needs to be run on multiple systems, each combination and system needs to be defined as an app. Code that can be forked at the command line or submitted to a batch scheduler can be registered as an Agave app and run through the Jobs service. The user sends a request by filling in the application inputs and job details on CEP portal. The app is packaged with 3 files:</p>
+  <p>Applications are executable code available for invocation through the TAPIS Jobs service on a specific execution system. If a single code needs to be run on multiple systems, each combination and system needs to be defined as an app. Code that can be forked at the command line or submitted to a batch scheduler can be registered as a TAPIS app and run through the Jobs service. The user sends a request by populating the application inputs and job details in the portal. Applications are packaged together using three files:</p>
 
   <dl class="s-inline-dl  u-nested-text-content">
-    <dt>App.json</dt><dd>JSON file with application definitions and parameters such as (name, version, label).</dd>
-    <dt>Wrapper.sh</dt><dd>wrapper file, shell script which executes the job on HPC/Docker.</dd>
-    <dt>Test.sh</dt><dd>runs a test before the job is launched and cleans up the files.</dd>
+    <dt>App.json</dt><dd>A JSON file with application definitions and input parameters (i.e. - name, version, label).</dd>
+    <dt>Wrapper.sh</dt><dd>A shell script which executes the job on the targeted resource (i.e. - an HPC system or virtualized container).</dd>
```

Comment:
I haven't seen "i.e. - " before...

```suggestion
    <dt>App.json</dt><dd>A JSON file with application definitions and input parameters (i.e. name, version, label).</dd>
    <dt>Wrapper.sh</dt><dd>A shell script which executes the job on the targeted resource (i.e. an HPC system or virtualized container).</dd>
```

Reply:
ahh, the dash.... good catch!